src/style.css

```diff
@@ -62,24 +59,20 @@ p {
 }
 
 .stars__stars {
-  background-image: url(./images/star-active.svg);
-  background-repeat: no-repeat;
+  background: url(./images/star.svg) no-repeat center;
   width: 16px;
   height: 16px;
   margin-right: 4px;
-  background-position: center;
 }
 
-.stars__stars:nth-child(n+5) {
-  background-image: url(./images/star.svg);
+.stars__stars:nth-child(-n+4) {
```

Comment:
not fixed, it should be possible to change quantity of stars with modifier
<img width="827" alt="image" src="https://user-images.githubusercontent.com/53611910/205524333-4446cfe8-7f92-4fbb-b978-842e0dc2b8a3.png">